src/app/core/models/issue.model.ts

```diff
@@ -92,7 +97,7 @@ export class Issue {
     issue.githubComments = githubComments;
     issue.teamAssigned = teamData;
     issue.issueComment = template.comment;
-    issue.teamResponse = template.teamResponse !== undefined ? template.teamResponse.content : undefined;
+    issue.teamResponse = template.teamResponse !== undefined ? Issue.updateTeamResponse(template.teamResponse.content) : undefined;
```

Comment:
With this change, am I right to say that if the team response is erased on GitHub (i.e. someone deletes the response in the comment), then the app will still display the response as `No details provided by team` ?

One example of such a comment:
![image](https://user-images.githubusercontent.com/35621759/75677152-c7e1d400-5cc5-11ea-8132-6b36786b75ac.png)

Reply:
Yeap.